assets/base.css

```diff
@@ -155,7 +155,8 @@
   --color-icon: rgb(var(--color-base-outline-button-labels));
 }
  
-.product-card-wrapper .card {
+.product-card-wrapper .card,
+.contains-product-card .slider__slide {
```

Comment:
Good catch Ludo. I just had to get the latest from `main`

Reply:
We already have `contains-card--standard` and `contains-card--card` so I think since theyre different modifiers it wouldnt make sense

Reply:
Oof 😅 
- `contains-card`
- `contains-card--standard`
- `contains-card--collection`
- `contains-card--product`
- `contains-card--article`

Gotta be done though.

Reply:
I don't mind it personally. They're all variations of the same thing 👍

Reply:
Yeah that's expected. They should all theoretically have a `contains-card` OR `contains-card--standard` AND one of the `--article`, `--product`, OR `collection`. 

Though if I recall, `contains-card` was only added as because it would sense for it to exist with `--standard` being a thing. That still remains true, but I don't think it's actually targeted in the styles anywhere so it could be feasibly removed (in the future if not now) to reduce some redundancy. Any thoughts on that @ludoboludo? I actually think it was you who wanted me to add that initially.

Reply:
Interested in how @ludoboludo feels as `contains-product-card` was from a PR of his, but I'm fine updating the main-product instance to use the new class, as the pattern is already established with `--standard`. We can just add the `contains-card` class to main-product as well if we really want to.